java/java.j2seembedded/build.xml

```diff
@@ -25,7 +25,7 @@
 
     <target name="probe" depends="init">
         <mkdir dir="build/probe/classes"/>
-        <javac srcdir="jreprobe" destdir="build/probe/classes" source="1.3" debug="true" deprecation="true" target="1.1"/>
+        <javac srcdir="jreprobe" destdir="build/probe/classes" source="1.6" debug="true" deprecation="true" target="1.6"/>
```

Comment:
I think cases like this one are the tricky part: the probe will (AFAIK) run on the target JDK (i.e. on the JDK the user is developing against). So this effectively means NB won't recognize JDK 5 and older. How huge problem is that is a matter of discussion I guess.